src/js/footer/featureFlagLiveChat/index.js

```diff
@@ -0,0 +1,24 @@
+// Import Firebase SDK modules
+import { initializeApp } from "https://www.gstatic.com/firebasejs/10.8.1/firebase-app.js";
+import {
```

Comment:
Also in my opinion we better to make this a module which accepts parameters to return different values from firebase. this way we can reuse this module for other flags as well.

Reply:
sure will do that